node/impl/full/state.go

```diff
@@ -139,15 +139,10 @@ func (m *StateModule) StateMinerInfo(ctx context.Context, actor address.Address,
 		return miner.MinerInfo{}, xerrors.Errorf("failed to load miner actor state: %w", err)
 	}
 
-	// TODO: You know, this is terrible.
-	// I mean, we _really_ shouldn't do this. Maybe we should convert somewhere else?
```

Comment:
:partying_face:

Reply:
yeah this is the happiest outcome of it all

Reply:
🙇